asserts/account.go

```diff
@@ -58,7 +58,7 @@ func (acc *Account) DisplayName() string {
 }
 
 // Validation returns the level of confidence of the authority in the
-// account's identity, expected to be "unproven" or "verified", and
+// account's identity, expected to be "unproven", "verified" or "starred", and
```

Comment:
I would swap "starred" or "verified" here.   All the stuff after the ", and ..." wasn't used so far, and now it's not true so should be removed from the comment.